### PR TITLE
Create # A Guide to Markdown.md

### PR DESCRIPTION
**Pull request by micmarti85**
A Guide to Markdown

Signed-off-by: micmarti85 <91504054+micmarti85@users.noreply.github.com>

### CODE REVIEW

**Comment by dplanella** [comment]
@micmarti85 thanks for adding the content! You asked me about audience elsewhere (good point), and I should probably add a note about that on the handbook itself.

The audience for the handbook is us, the Developer Experience and Community Success team. The content on the handbook should help us document existing processes and define new ones that help us to do our daily work. It should be a reference resource we can refer to or point team mates to when questions come up.

In terms of the actual contribution, I would argue that it might better fit into the handbook as a brief quickstart guide to Markdown. Rather than a full guide, which are already available online https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.

In summary, I'd rather see a short "get started" guide with the basics, which then points to the [GitHub Markdown guide](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax).

I'd be curious to hear what the rest of the team thinks, though.

**Comment by jmertic** [comment]
I'd agree with that approach. Having this doc be our internal "best practices" would be solid here and then could point to the guides @dplanella referenced.

**Comment by dplanella** [comment]
Closing as per review comments. Thank you for your work in this PR.
